database/migrations/2019_10_29_182148_create_sites_table.php

```diff
@@ -20,7 +20,7 @@ public function up()
             $table->unsignedBigInteger('county_id')->nullable();
             $table->string('name')->index();
             $table->string('owner_name')->index()->nullable();
-            $table->text('owner_address')->nullable();
+            $table->text('owner_contact')->nullable();
```

Comment:
You need to only rename the column in a new migration. Editing an old migration doesn't work. This will actually break installing new sites. So let's revert this and keep the new migration you created.
```suggestion
            $table->text('owner_address')->nullable();
```